dependency-graph/dependency-review.yml

```diff
@@ -0,0 +1,11 @@
+name: 'Dependency Review'
```

Comment:
I see a new directory has been created for `dependency-graph`. While I agree that `code-scnaning` directory does not make sense but we also would not want the directories to proliferate.
@anuragc617 @Phantsure would it be okay if we rename `code-scanning` dir to `security` and keep both together.

Reply:
@elireisman for now keep the DR starter workflow inside `code-scanning` directory. This will allow it to get shipped. And then we can follow up renaming directory to `security` and updating the UI to work with this directory.

Reply:
thanks - just moved my action spec and props files into the `code-scanning/` and `code-scanning/properties` dirs. Let me know if there's more to do there related to this move 👍